Design rules/Policy Design Rules.md

```diff
@@ -0,0 +1,70 @@
+# Policy Design Rules
+
+In dit document beschrijven we hoe we in het Kenniscentrum Standaarden van VNG Realisatie omgaan met Design Rules. 
+Daarbij doelen we op de eigen Design Rules, de Design Rules die in de Nederlandse API Strategie zijn beschreven maar ook op de Extensions die op beide niveaus beschikbaar zijn.
```

Comment:
Hebben wij binnen deze kennisbank ook Extensions? Waar staan die...

Reply:
Die zijn er nog niet maar het zou zo maar kunnen dat een aantal van de issues met het label 'Design Rule' als VNG-R Extension worden benoemd. De Policy voorziet dus in een situatie die zich mogelijk voor kan doen.